src/config/schema.rs

```diff
@@ -52,30 +53,6 @@ pub struct SeafowlConfig {
     pub misc: Misc,
 }
 
-#[derive(Deserialize, Debug, PartialEq, Eq, Clone)]
-#[serde(tag = "type", rename_all = "snake_case")]
-pub enum ObjectStore {
-    Local(Local),
-    #[serde(rename = "memory")]
-    InMemory(InMemory),
-    #[cfg(feature = "object-store-s3")]
-    S3(S3),
-    #[cfg(feature = "object-store-gcs")]
```

Comment:
We should probably get rid of these two features in Cargo.toml (not used in the new configs, and I don't think we ever opt-out of these).

Reply:
I purged also some occurrences of it in the code.